src/components/people.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useContext, useEffect } from 'react';
+import { getPeople } from '../api';
+import { MyContext } from './state';
+import { Loader } from './Loader';
+import { Error } from './error';
+import { Person } from './person';
+import { Person as PersonType } from '../types';
+
+export const People = () => {
+  const { person, setPerson, error, setError, setIsLoading, isLoading } =
+    useContext(MyContext);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(fetchedPeople => {
+        setPerson(fetchedPeople);
+      })
+      .catch(() => {
+        setError('Unable to load people: ');
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      {isLoading ? (
+        <Loader />
+      ) : error ? (
+        <Error />
+      ) : person.length === 0 ? (
```

Comment:
The condition `person.length === 0` checks if there are no people, but since 'person' is expected to be an array of people, the variable name should be pluralized to 'people' to avoid confusion.